graphium/config/_loader.py

```diff
@@ -455,6 +455,14 @@ def load_accelerator(config: Union[omegaconf.DictConfig, Dict[str, Any]]) -> Tup
     merge_dicts(config, config_override)
     accelerator_type = get_accelerator(config_acc)
 
+    if accelerator_type == "gpu":
+        precision = config_acc.get("float32_matmul_precision", None)
+        if precision is not None:
+            torch.set_float32_matmul_precision(precision)
+
+    if accelerator_type != "ipu":
+        del config["predictor"]["optim_kwargs"]["loss_scaling"]
```

Comment:
The config override doesn't work for deleting keys (and I didn't find a straightforward fix). `loss_scaling` seems to be a argument specific to Graphcore optimizers

Reply:
Ahhh I see. That's a problem. I was thinking of creating a `config_update` and `config_override` that would have different behaviours, but then it makes things very complicated.

I don't like your current solution. But I guess we can proceed with the PR and open a new issue #404  (Nice issue number 😄 )